src/aim/pytorch/hub_mixin.py

```diff
@@ -0,0 +1,45 @@
+"""Illustrates the mixin for one class."""
```

Comment:
I think this example can be either added to `README.md` or let's create `examples` folder. In any case, I don't think it should be placed inside the package itself.